src/components/pipelines-overview/PipelinesOverviewPage.tsx

```diff
@@ -13,11 +13,14 @@ import TimeRangeDropdown from './TimeRangeDropdown';
 import RefreshDropdown from './RefreshDropdown';
 
 interface PipelinesOverviewPageProps {
-  match: Rmatch<never>;
+  match: Rmatch<{ ns?: string }>;
 }
 
-const PipelinesOverviewPage: React.FC<PipelinesOverviewPageProps> = () => {
+const PipelinesOverviewPage: React.FC<PipelinesOverviewPageProps> = ({
+  match,
+}) => {
   const { t } = useTranslation('plugin__pipeline-console-plugin');
+  const namespace = match.params?.ns ? match.params.ns : '-';
```

Comment:
The check is added but it still  gives error for params is undefined and failing to load the UI